test/integration/load/load_test.go

```diff
@@ -41,7 +44,7 @@ In order to run the scale tests, you need a k8s cluster and its kubeconfig.
 If no kubeconfig is passed, the test will attempt to find one in the default location for kubectl config.
 Run the tests as follows:
 
-go test -timeout 30m -tags load -run ^TestLoad$ -tags=load
+go test -timeout 30m -tags load -run ^TestLoad$
```

Comment:
Since the file still has `build` tag we will need `-tags` right ?

Reply:
Yeah but it's already there, just before , in that same line